src/AsyncEventSource.cpp

```diff
@@ -253,7 +253,7 @@ size_t AsyncEventSource::count(){
 }
 
 bool AsyncEventSource::canHandle(AsyncWebServerRequest *request){
-  if(request->method() != HTTP_GET || !request->url().equals(_url))
+  if(request->method() != WebRequest::Method::HTTP_GET || !request->url().equals(_url))
```

Comment:
sorry have not looked in a long time, but how about it is renamed to WebMethod instead of WebRequest::Method